### PR TITLE
#379 bug/Add missing open new file button in TabPanelContainer

### PR DESCRIPTION
**Pull request by MelindaSW**
Added a PivotItem with index="open-new-file-tab" and a check in the tabOnClick function where showOpenDialog() is called when index === 'open-new-file-tab.

### CODE REVIEW

File: src/js/view/components/TabPanelContainer.js [hunk 1]
```diff
@@ -135,6 +139,13 @@ function TabPanelContainer(props) {
             ></PivotItem>
           );
         })}
+
+        <PivotItem
```
Comment (Doverstav) [inline src/js/view/components/TabPanelContainer.js]: I'm a bit unsure about using a `PivotItem` as an actionButton. While it looks good and works fine, looking at the [documentation](https://developer.microsoft.com/en-us/fabric#/controls/web/pivot) it says that 

> Tapping on a pivot item header navigates to that header's section content.

Thus, I feel that `PivotItem`s should be used for navigating to content, and not to take actions in the application.  

Instead, I think something like an `IconButton` or an `ActionButton` could work, something like this:

```suggestion
<IconButton
    iconProps={{ iconName: 'Add' }}
    onClick={() => {
        showOpenDialog();
    }}
></IconButton>
```

This should be added inside the `Stack` component, and not in the `Pivot` component. To fix alignment, we would have to add `verticalAlign="center"` to the `Stack` as well. This `IconButton` would also have a bluish tint instead of the greyish one from your solution (which I think looked better), but perhaps that could be changed if we feel strongly about the looks of the button.

By adding `onClick` directly to the button, we would also avoid adding "complications" to the `tabOnClick()` function.

In summary, looks good and works great, but I'm unsure if the correct component have been used.

Comment (MelindaSW) [reply src/js/view/components/TabPanelContainer.js]: Thank you for the feedback! Had a feeling that there was something off with my solution but could not put my finger on it. I will take a look at the IconButton/ActionButton approach and make some changes.
